open_alchemy/__init__.py

```diff
@@ -250,17 +251,26 @@ def build_json(
     spec_filename: str,
     package_name: str,
     dist_path: str,
+    format_: PackageFormat = PackageFormat.NONE,
 ) -> None:
     """
     Create an OpenAlchemy distribution package with the SQLAlchemy models.
 
     The package can be uploaded to, for example, PyPI or a private repository for
     distribution.
 
+    The formats can be combined with the bitwise operator or (``|``), for
+    instance, building both sdist and wheel packages can be specified like that:
+
+    .. code-block: python
+
+        format_ = PackageFormat.SDIST|PackageFormat.WHEEL
+
     Args:
         spec_filename: filename of an OpenAPI spec in JSON format
         package_name: The name of the package.
         dist_path: The directory to output the package to.
+        format_: The format(s) of the archive to build if any
```

Comment:
Please note that it is an optional argument

Reply:
Addressed ✅